### DISCUSSION TITLE
TOC not created for Blog post

### DISCUSSION THREAD

**Discussion by Techbrunch** [squidfunk/mkdocs-material#4391] [Q&A] [answered]
What setting should be enabled to have the TOC for blog posts ?

It works for regular pages but not for a blog post.

**Comment by Techbrunch** [comment, marked as answer]
Ok I found the issue, my post had the following titles:

```
# One
# Two
### Three
```

Apparently this broke the TOC but I don't think there was a warning. @squidfunk Should I create an issue ?

**Reply by squidfunk** [reply]
No, there's no warning – Material for MkDocs only supports one `h1` for each Markdown document, but since we're only a theme, we can't issue a warning for all documents. We could implement this for blog posts, but not for all documents, which doesn't make any sense.
